pkg/kubelet/kubelet_pods.go

```diff
@@ -1974,6 +1974,13 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 			} else {
 				determineResource(v1.ResourceCPU, container.Resources.Limits, oldStatus.Resources.Limits, limits)
 			}
+			// if the cpu limit is set less than 2m, the minimum value set will 10m,
```

Comment:
This comment implies that if the cpu limit is 3m, the reflected minimum value will still be 3m but if the cpu limit is 1m, the minimum value will be 10m.

Reply:
That was a mistake, For limit its 10m and for request its 2m.

Reply:
Thank you for the fix,